doc/Community-Build/Building-CORTX-From-Source-for-SingleNode.md

```diff
@@ -0,0 +1,63 @@
+# Setting up the CORTX Environment for Single Node
+
+You can deploy the CORTX stack from the source code. This document provides step-by-step instructions to deploy and configure the CORTX environment for a single-node Virtual Machine (VM).
+
+The CORTX deployment and configuration is a four-step procedure:
+
+-   Generate all the CORTX service packages such as Motr, S3Server, HA, HARE, Manager, Management-portal, etc. To know more about CORTX components, see [CORTX Components guide](https://github.com/Seagate/cortx/blob/main/doc/Components.md).
+-   Deploy the generated packages to create and run the CORTX cluster.
+-   After the CORTX cluster is running, configure the CORTX GUI to communicates with different CORTX components and features.
+-   Configure the S3 account to perform the IO operations.
+
+## Prerequisites
+
+-   Create a [CentOS 7.8.2003](http://repos-va.psychz.net/centos/7.8.2003/isos/x86_64/) VM with following minimum configuration:
+
+    - RAM: 8GB
+    - Processor: 4
+    - NICs: 3
+    - OS Disks: 20GB
+    - Data Disks: 8 disks of 10GB
+    - Metadata Disks: 1 disk of 10GB
```

Comment:
Changes as per,

- OS Disks: 1 disk of 20GB
- Data Disks: 8 disks of 10GB
- Metadata Disks: 1 disk of 8GB